### ISSUE TITLE
migration from libwebrtc 65 to libwebrtc 73

### ISSUE THREAD

**Issue by ludocosmo** [CoSMoSoftware/OBS-studio-webrtc#67]
Update code so that OBS with webrtc will make use of libwebrtc m73 instead of libwebrtc 65.

* [x] Mac
* [x] Winddows (no CI)
* [ ] Linux

**Comment by agouaillard-cosmo** [comment]
@ludocosmo, can you assign the right person there?

**Comment by agouaillard-cosmo** [comment]
blocked on #79 #84

**Comment by agouaillard-cosmo** [comment]
linux is blocked on some deeper problem. I created a separate ticket for it (#84). I will close this, and keep the linux work ongoing on the other ticket.
